### PR TITLE
Update to Drush 9.3.0 and Drush 8.1.17 and convert to Circle 2.0

### PR DESCRIPTION
**Pull request by greg-1-anderson**
There were [very few changes from 8.1.16 to 8.1.17](https://github.com/drush-ops/drush/releases/tag/8.1.17), but updating to 8.1.17 at the same time because some of the fixes were for notices in php 7.1 and 7.2.

Also updating to Circle 2, as the Circle 1 build was broken. (c.f. https://getpantheon.atlassian.net/browse/ENG-456)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,9 +12,14 @@ deps-macos:
 
 deps-circle:
 	sudo apt-get -y install rpm
-	gem install package_cloud
+	# gem install package_cloud
```
Comment (jslang) [inline Makefile]: Did you mean to remove this, or is comment purposeful?

Comment (greg-1-anderson) [reply Makefile]: That's dead code - that is no longer used with the Circle 2 build, which uses docker containers with our dependencies pre-installed. I forgot to delete the rule -- I'll do that now.

File: scripts/build-rpm.sh [hunk 2]
```diff
@@ -3,101 +3,49 @@
...
+  version=$(echo $version_with_datecode | cut -d : -f 1)
+  iteration=$(echo $version_with_datecode | cut -d : -f 2)
 
 	version_untampered=$version
 	version=$(echo $version | sed -e 's/\([^-]*\)-\([^-]*\)-\(.*\)/\1.\2.0\-\3/')
 
 	releasenum=${version%%.*}
 	name="$shortname$releasenum"
-	iteration="$(date +%Y%m%d%H%M)"
+	#iteration="$(date +%Y%m%d%H%M)"
```
Comment (jslang) [inline scripts/build-rpm.sh]: Could clean this up too.

**Review by jslang — APPROVED** [review]
LGTM

**Pull request merged**